Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:2.7.8
+MAINTAINER Seigo Uchida <spesnova@gmail.com> (@spesnova)
+
+RUN pip install locustio pyzmq
+RUN mkdir /test
```

Comment:
As far as I know `WORKDIR /test` instruction makes directory, so you don't need this line :point_up: